README.md

```diff
@@ -3,25 +3,49 @@ Azure Functions prepared for the Mapping Viz Tools project. This repo contains [
+ `\include path\to\repo\mapping-viz-functions\db\data\test_data.sql`  
+See more on how to use the `\include` command in the [psql documentation](https://www.postgresql.org/docs/9.5/app-psql.html)). 
+
+### `yarn install`
 To install the necessary dependencies
 
-### yarn test
+### `yarn test`
 To test the app
 
-### yarn start
+### `yarn start`
 To start the server. You an then try hitting the available endpoints (see terminal).
 
 ### Connection Strings
 For security, connection strings are set on the Azure Portal. Check out the `PG_DB_CONNECTION_STRING` ENV variable for this function when logged into the portal. For local testing, this variable can be set in the `local.settings.json` file. Just be careful not to commit any sensitive information when testing. 
 
-### Deployment
+## Deployment
 Deploying an Azure Function is simple, using the Azure VS Code extension. You can follow steps [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-function-vs-code?pivots=programming-language-javascript#publish-the-project-to-azure) on how to publish a function. 
 
 #### Staging or Production?
-Although you could use Azure [Slots](https://docs.microsoft.com/en-us/azure/azure-functions/functions-deployment-slots) to handle different environments, you can also deploy the same function at a different location (eg. `mapping-viz-functions-staging` vs `mapping-viz-functions`) and set a different connection string at each endpoint to handle staging and production databases.
\ No newline at end of file
+Although you could use [Azure Slots](https://docs.microsoft.com/en-us/azure/azure-functions/functions-deployment-slots) to handle different environments, you can also deploy the same function at a different location (eg. `mapping-viz-functions-staging` vs `mapping-viz-functions`) and set a different connection string at each endpoint to handle staging and production databases.
+
+## Database Maintenance
+The current schema is saved in db/schema.sql. This files are generated using [pg_dump](https://www.postgresql.org/docs/9.5/app-pgdump.html) (version 9.5.23). There are also files containing test data for local development, and the full dataset for back-up purposes - those live in db/data. The full_data.sql file is compressed using the `pg_dump` command's built in compression (using the flag `--compress=9`).
+
+We have currently decided to go with a bare-bones approach to database maintenance and schema migrations, due to the fact that we do not foresee the schema changing much going forward. If the trajectory of the project changes, it may be helpful to make use of a package or library to manage the database.
+
+### Schema Migrations
+The current process for updating the schema is as follows:
+1. Add a .sql file named 'migration10-13-20.sql' (replace with current date) to the db/schema_migrations folder. This file should contain the SQL commands that need to be executed in order to make the schema change, e.g. `ALTER TABLE`, etc.
```

Comment:
I'd add an empty placeholder directory for `./db/schema_migrations`

Reply:
I had created one, but looks like it didnt get committed because git doesn't allow empty directories - https://git.wiki.kernel.org/index.php/GitFaq#Can_I_add_empty_directories.3F

Reply:
You can use an empty file named `.keep` in the directory to allow git to track it.  See here for an example on another zooniverse repo: https://github.com/zooniverse/zoo-stats-api-graphql/tree/master/tmp

Reply:
yeah - that @lcjohnso said. I usually use `touch ./dir_i_want_to_keep/.gitkeep` as my filename

Reply:
nice. will do thanks for the tip 👍